forward_script/src/PageInformationCollector.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace Wikimedia\ForwardScript;
+
+use Mediawiki\Api\MediawikiApi;
+use Mediawiki\Api\SimpleRequest;
+use Symfony\Component\Process\Process;
+use Symfony\Component\Process\Exception\RuntimeException;
+use Wikimedia\ForwardScript\ApplicationException;
```

Comment:
This import statement is not required here (not meaning it is wrong). As `PageInformationColletor` is in the same namespace as `ApplicationException`, the unqualified exception class name would be resolved properly.